js/app.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+let resourceButton = document.getElementById('requestResourceButton');
+resourceButton.addEventListener('click', giveInformation);
+
+function giveInformation() {
+  let inputBox = document.getElementById('resourceId').value;
+  let inputSelector = document.getElementById('resourceType').value;
+  let infoBox = document.getElementById('contentContainer');
+
+  if (inputSelector === 'people') {
+    const getPeople = new XMLHttpRequest();
+    getPeople.addEventListener('load', reqListenerPeople);
+    getPeople.open('GET', 'https://swapi.co/api/people/' + inputBox);
+    getPeople.send();
+  }
+
+  function reqListenerPeople() {
+    if (this.status !== 200) {
```

Comment:
Since all your errors across the three types are the same, you could pull that out into a function instead of copying and pasting it twice.